LICENSE.txt

```diff
@@ -123,25 +110,28 @@ http://msdn.microsoft.com/en-us/library/gg405489(PandP.40).aspx
 
 ### Python Standard Library
 https://www.python.org/
-https://www.python.org/download/releases/2.7/license/
```

Comment:
we are still shipping the 2.7 std lib and so I think this should remain.

Reply:
Actually, the link I replaced this with includes the license of every Python version up to this day. It's an updated version of the old link, which included the license of every Python version up to 2.7.